app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java

```diff
@@ -0,0 +1,364 @@
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+					}
+				});
+
+		Date now = new Date();
+		Calendar cal = Calendar.getInstance();
+		cal.setTime(now);
+		for (int i = 0; i < 7; i++) {
+			// Formatting for each day
+			SimpleDateFormat dateFormat = new SimpleDateFormat("EEE");
+			dateFormat.setCalendar(cal);
+			String sDay = dateFormat.format(cal.getTime());
+
+			// Formatting for each date
+			int date = cal.get(Calendar.DAY_OF_MONTH);
+			String sDate = Integer.toString(date);
+			SpannableString ssDate = new SpannableString(sDay + '\n' + sDate);
+			ssDate.setSpan(new RelativeSizeSpan(0.8f), 0, 3, 0);
+			ssDate.setSpan(new RelativeSizeSpan(2f), 4, ssDate.length(), 0);
+			TextView tv = dateList.get(i);
+			tv.setText(ssDate);
+
+			cal.add(Calendar.DAY_OF_YEAR, 1);
+		}
+
+		breakfastText.setOnClickListener(
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+
+						mealType = MealType.BREAKFAST;
+						changeListAdapter(mealType, presenter.getSelectedDate());
+					}
+				});
+
+		lunchText.setOnClickListener(
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+
+						mealType = MealType.LUNCH;
+						changeListAdapter(mealType, presenter.getSelectedDate());
+					}
+				});
+
+		dinnerText.setOnClickListener(
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						breakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						lunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						dinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+
+						mealType = MealType.DINNER;
+						changeListAdapter(mealType, presenter.getSelectedDate());
+					}
+				});
+		LocalDateTime currentTime = LocalDateTime.now();
+
```

Comment:
i'd rather have the newline before `237` than afterwards, but that's a minor comment